modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -505,7 +503,18 @@ export default {
       }
     },
     async refresh(options = {}) {
-      let url = window.location.href;
+      const refreshable = document.querySelector('[data-apos-refreshable]');
+      if (options.scrollcheck) {
+        window.apos.adminBar.scrollPosition = {
+          x: window.scrollX,
+          y: window.scrollY
+        };
+      }
+
+      if (!refreshable) {
```

Comment:
I just moved the logic here, to avoid to perform an http request when there is no refreshable element. tiny perf improvement.

Reply:
I kept the exact same logic, so if there is not refreshable element, it still call:
```
apos.bus.$emit('refreshed');
this.rememberLastBaseContext();
```
As it was before, not sure if it's usefu in this context though.